core/templates/pages/exploration-editor-page/editor-tab/exploration-editor-tab.component.html

```diff
@@ -100,6 +99,9 @@ <h4>Now for the fun part...</h4>
       margin-right: 0;
       width: 100%;
     }
+    .exp-state-version-history {
```

Comment:
What's the rationale for this class?

Reply:
Just like the "exploration overview" section is not visible in mobile screens, I think we should also hide this "version history" section for mobiles.